modules/datasets/common/dataset.py

```diff
@@ -63,7 +63,7 @@ def _download(self):
             path = wget.download(self.url, self.path)
             # extract the dataset.
             with zipfile.ZipFile(path, 'r') as zip_file:
-                zip_file.extractall(self._get_extract_path())
+                zip_file.extractall(os.path.splitext(path)[0])
```

Comment:
It is better to implement `Dataset._get_extract_path` as following,
```python
def _get_extract_path(self):
    return os.path.join(self.path, self.name)
```
because the path structure should be like
```bash
$ tree -d 1 orig_data/
1 [error opening dir]
orig_data/
├── lsp_dataset
│   ├── images
│   └── visualized
└── lspet_dataset
    └── images

5 directories
```